frontend/src/pages/DevicesPage/DevicesPage.tsx

```diff
@@ -37,8 +45,8 @@ export const DevicesPage = () => {
           <div className={css.header}>
             <SearchField />
             <AccountSelect />
-            <FilterButton />
             <RefreshButton />
+            <FilterButton handleDrawer={handleDrawer} open={open} />
```

Comment:
Could also be called onOpenDrawer....
```suggestion
            <FilterButton onOpen={onOpen} open={open} />
```